include/pass/shrink_for.h

```diff
@@ -40,21 +40,29 @@ class ShrinkFor : public CompTransientBounds<SymbolTable<Mutator>> {
     std::vector<Var> iterStack_;
     std::vector<std::unordered_set<std::string>> namesStack_;
 
+    Stmt subAST_;
+    std::unordered_map<StmtSeq, Stmt> subASTInSeq_;
```

Comment:
It seems there is no need to record a `StmtSeq` to `Stmt` mapping. Just record a est of `Stmt` to be checked, and check it in `visitStmt`.